mobx_examples/lib/connectivity/connectivity_store.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:mobx/mobx.dart';
```

Comment:
I think it would be good to have an example page, say `examples/connectivity` for [mobx.pub](https://mobx.pub) too. What do you think?

Reply:
Can do though I suspect it'll be rather brief. Another issue is with the delay set to four seconds, I suspect not a lot of users will wait to see how it looks if we put a gif on there.

In any case I think that could be done in another PR. I did see that you have another PR for updating the site as well. Not sure if you plan to complete the soon or is that still in-progress?

Reply:
That PR was for upgrading the Docz tool. We can do the docs in a separate PR for the example